virtual_rainforest/main.py

```diff
@@ -57,11 +57,7 @@ def select_models(model_list: list[str]) -> list[Type[BaseModel]]:
     LOGGER.info("Attempting to configure the following models: %s" % unique_models)
 
     # Make list of missing models, and return an error if necessary
-    miss_model = [
-        model
-        for model in unique_models
-        if model not in MODULE_REGISTRY or "model" not in MODULE_REGISTRY[model]
-    ]
+    miss_model = [model for model in unique_models if model not in MODULE_REGISTRY]
     if miss_model:
```

Comment:
I think this should do the trick and is faster:

```suggestion
    if any(model not in MODULE_REGISTRY for model in unique_models):
```

Reply:
But we use the missing names list in the logging output.